app/api/public-cloud/aws-roles/helpers.ts

```diff
@@ -56,17 +95,19 @@ async function getPublicCloudProjectUsers(searchLicencePlate: string): Promise<R
 
   if (project?.projectOwnerId) {
     const projectOwner = await getUserById(project.projectOwnerId);
```

Comment:
I know, it is reviewed already before, but can we use `include` clause of `prisma` query to avoid the additional database trips?

Reply:
yes, that is much better, changed